src/Visitor/MongoOdm.php

```diff
@@ -73,6 +78,26 @@ public function visit(OperationInterface $operation)
     }
 
     /**
+     * Provides the  Doctrine QueryBuilder
+     *
+     * @return QueryBuilder
+     */
+    public function getBuilder()
+    {
+        return $this->queryBuilder;
+    }
+
+    /**
+     * Provides the Doctrine Query object to execute.
+     *
+     * @return \Doctrine\ODM\MongoDB\Query\Query
+     */
+    public function getQuery()
```

Comment:
getQuery() is used here:
https://github.com/libgraviton/graviton/pull/68/files#diff-16e06489fc38bdfb5fcba04f42544688R230